pipelines/matrix/conf/test/globals.yml

```diff
@@ -1,4 +1,4 @@
-run_name: test-run
+run_name: ${oc.env:RUN_NAME,test-run}
```

Comment:
For sure not straight forward stuff! Really nice this all came together.

I'd love for us to:

- Add a little bit of documentation
- Discuss next steps, e.g., get rid of releases concept as we can now branch off